stdlib/public/Distributed/DistributedActor.swift

```diff
@@ -83,6 +86,27 @@ public protocol DistributedActor: AnyActor, Identifiable, Hashable, Codable {
     nonisolated var id: AnyActorIdentity { get }
 }
 
+@available(SwiftStdlib 5.5, *)
+extension DistributedActor {
+
+  public static func resolve<Identity>(_ identity: Identity, using transport: ActorTransport)
+      throws -> Self where Identity: ActorIdentity {
+    switch try transport.resolve(AnyActorIdentity(identity), as: Self.self) {
+    case .resolved(let instance):
+      return instance
+
+    case .makeProxy:
+      // FIXME: this needs actual implementation of distributedActorRemoteCreate
+      let remote: Any = distributedActorRemoteCreate(identity: identity, transport: transport)
+      return remote as! Self
+    }
+  }
+}
```

Comment:
I think we'll actually get away with just implementing it like that...

It depends how we'd want to make it available for protocols later on, but that's "just" a typechecking change, the implementation is always the same :-)